aptos-move/framework/aptos-experimental/sources/trading/order_book/bulk_order_book_types.move

```diff
@@ -0,0 +1,617 @@
+/// # Bulk Order Book Types Module
+///
+/// This module defines the core data structures and types used by the bulk order book system.
+/// It provides the foundational types for representing multi-level orders, order results,
+/// and match operations.
+///
+/// ## Key Data Structures:
+///
+/// ### 1. BulkOrder
+/// Represents a multi-level order with both bid and ask sides. Each side can have multiple
+/// price levels with associated sizes.
+///
+/// ### 2. BulkOrderResult
+/// Represents the result of an order matching operation, containing details about the
+/// matched order including price, size, and remaining quantities.
+///
+/// ### 3. SingleBulkOrderMatch
```

Comment:
Can you elaborate a bit more on what's the difference between BulkOrderResult and SingleBulkOrderMatch?

Reply:
Merged into OrderMatchResult